pom.xml

```diff
@@ -65,6 +65,18 @@
             <version>4.7.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>annotations</artifactId>
+            <version>3.0.1</version>
+        </dependency>
+
+        <!-- https://mavenlibs.com/maven/dependency/com.infradna.tool/bridge-method-injector -->
```

Comment:
Is there any motivation to leave this line as a comment?

Reply:
No, I  will remove it.